xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -692,6 +699,8 @@ void CDRMUtils::DestroyDrm()
   }
 
   m_renderFd.reset();
+
+  CSessionUtils::Close(m_fd);
```

Comment:
Having to do this defeats the point of using the RAII helper for fds in the first place. I think it would be better to either modify it so that it actually is RAII or stop using the helper.

Reply:
Yea, in this case it's probably better to create a new class that combines the logind stuff and the CFileHandle stuff